deploy/handler/operator.yaml

```diff
@@ -42,81 +40,74 @@ spec:
           env:
             - name: WATCH_NAMESPACE
               value: ""
-            - name: RUN_WEBHOOK_SERVER
-              value: ""
             - name: POD_NAME
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: RUN_WEBHOOK_SERVER
+              value: ""
             - name: OPERATOR_NAME
-              value: "{{template "handlerPrefix" .}}nmstate-handler"
-            - name: NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
-            - name: INTERFACES_FILTER
-              valueFrom:
-                configMapKeyRef:
-                  name: {{template "handlerPrefix" .}}nmstate-config
-                  key: interfaces_filter
-            - name: WEBHOOK_PORT
-              value: "54874"
+              value: "{{template "handlerPrefix" .}}nmstate"
```

Comment:
Why does the webhook deployment need to know the Operator name?

Reply:
Manager was failing operator-sdk need it (I don't remeber, also we need it to point to MutatingWebhookConfiguration)